tsconfig.json

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
This file was created using `tsc --init` so we have inline docs for most options. The Monitor-specific config I enabled can be seen in b813293163c6e2ab5a7d07659a1658e10e4be47c.